src/mappings/membership/index.ts

```diff
@@ -68,6 +70,11 @@ export async function processMemberProfileUpdatedEvent({
   }
 }
 
+function updateMemberHandle(member: Membership, newHandle: Uint8Array) {
+  member.handleRaw = '0x' + Buffer.from(newHandle).toString('hex')
```

Comment:
you can use the `u8aToHex` method from `polkadot/util`. It converts `Uint8Array` to hex string out of the box

```ts
import { u8aToHex } from '@polkadot/util'
```